programs/mpl-core/src/plugins/internal/permanent/permanent_freeze_delegate.rs

```diff
@@ -81,11 +81,13 @@ impl PluginValidation for PermanentFreezeDelegate {
         &self,
         ctx: &PluginValidationContext,
     ) -> Result<ValidationResult, ProgramError> {
-        if ctx.target_plugin.is_some() && self.frozen {
-            reject!()
-        } else {
-            abstain!()
+        if let Some(Plugin::PermanentFreezeExecute(stored)) = ctx.target_plugin {
```

Comment:
Isn't the permanent freeze delegate separate concern from the permanent freeze execute delegate?

Reply:
Yup I applied the change to the wrong file.